Gemfile

```diff
@@ -2,4 +2,4 @@ source "https://rubygems.org"
 gem 'github-pages', group: :jekyll_plugins
 gem "jekyll-include-cache"
 gem 'jekyll-archives'
-
+gem "webrick", "~> 1.7"
```

Comment:
Adding this dep and pinning ruby to 3.1.2 seems the way to reproducibly build this blog